build_env.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
We won't need this script if we could run the ``nodeenv -p --prebuilt`` command from tox. But this means installing node and npm every time that we run tox which is quite slow.

This script installs the node environment in the ``node_env`` folder the first time it is run, and then just re-uses it on subsequent calls.